plugins/action/git_publish.py

```diff
@@ -204,7 +219,11 @@ def _push(self: T) -> None:
             command_parts.extend(command_parameters)
             no_log[token_base64] = "<TOKEN>"
 
-        command_parts.extend(["push", "origin", "HEAD"])
+        tag = self._task.args.get("tag")
+        if tag:
```

Comment:
Can tags just be added to the end?  push, origin and HEAD could then be outside the condtion